dotCMS/src/main/java/com/dotmarketing/portlets/folders/action/OrderMenuAction.java

```diff
@@ -138,6 +139,28 @@ public void processAction(ActionMapping mapping, ActionForm form, PortletConfig
 		}
 	}
 
+	/**
+	 * Splits the parameter reorder_result and creates a map that contains the arranged list of items
+	 * @param req
+	 * @return
+	 */
+	private Map<String, String> getOrderedElements(ActionRequest req){
+
+		String reorderResult;
+		Map<String, String> orderedItems;
+
+		orderedItems  = new HashMap<>();
+		reorderResult = req.getParameter("reorder_result");
+
+		if (reorderResult != null){
+			for(String listItem:reorderResult.split("&")){
+				orderedItems.put(listItem.split("=")[0], listItem.split("=")[1]);

```

Comment:
Thanks Jonathan. It could be an improvement. I didn't include the validation because the listItem is created by us, so the input should be ok once this method is called